src/faldbt/project.py

```diff
@@ -132,24 +132,44 @@ def get_depends_on_nodes(self) -> List[str]:
         return self.node.depends_on_nodes
 
     def get_post_hook_paths(self, keyword: str = "fal") -> List[str]:
-        return self.meta.get(keyword, {}).get("post-hook", [])
+        meta = self.meta or {}
+
+        keyword_dict = meta.get(keyword) or {}
+        if not isinstance(keyword_dict, dict):
+            return []
```

Comment:
Question: what would be the case for `keyword_dict` being something else than `dict`? If it only happens when it is missing, maybe we could chain these and simplify the overall logic. 

Something like this maybe:
```python
return meta.get(keyword, []).get("post-hook", [])
```